sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusProcessErrorContext.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+/**
+ * Context for errors handled by the Service Bus processor.
+ */
+public class ServiceBusProcessErrorContext {
+    private final Throwable exception;
+    private final ServiceBusErrorSource errorSource;
+    private final String fullyQualifiedNamespace;
+    private final String entityPath;
+
+    ServiceBusProcessErrorContext(Throwable throwable, String fullyQualifiedNamespace, String entityPath) {
```

Comment:
Why do we need `fullyQualifiedNamespace` and `entityPath`? If this is needed for the error context, we should consider adding it to the `ServiceBusReceivedMessageContext` too? I know Event Hubs had these properties on both event and error context. So, we should consider adding it to both places.

Reply:
This is just mimicking what's already in .net, so this is more a question for @JoshLove-msft .